.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+    - "2.7"
+before_install:
+    - "export DISPLAY=:99"
+    - "sh -e /etc/init.d/xvfb start"
+install:
+    - "sh install_test_deps.sh"
+    - "python setup.py develop"
```

Comment:
This would not catch things like assets that aren't included in the package, right? To catch this type of issues with Travis, can you install the package instead of linking directly to the repository, like in https://github.com/open-craft/xblock-mentoring/blob/master/.travis.yml#L12 ?